contrib/storage-mongo/src/test/resources/datatype.json

```diff
@@ -0,0 +1 @@
+[{"_id": {"$oid" : "582081d96b69060001fd8938"}, "account" : { "$ref": "contact", "$id": {"$oid": "999cbf116b69060001fd8611"} }}, {"_id" : {"$oid" : "582081d96b69060001fd8939" } } ]
```

Comment:
Should we rename it to something like datatype_dbref.json? The name seems to suggest we have all datatypes in this document